### PR TITLE
Automatically block pi builds

### PR DESCRIPTION
**Pull request by DXCanas**
### Summary

Reintroducing automatically blocked pi builds here, with async enabled so that we can move on with the vision of auto-blocked builds.

### Reviewer guidance

- Build completes, even if pi image wasn't built.
- pi image hits a block step immediately. 

### References

#6494 enabled this
Retry of #6402 

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [x] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [x] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by lyw07** [comment]
is this PR still working in progress? I see that the build got canceled. Thank you!

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -31,8 +31,10 @@ steps:
 
   - trigger: "kolibri-raspbian-image"
     label: ":raspberry-pi:"
+    async: true
     build: &triggered_build_attributes
       message: "${BUILDKITE_MESSAGE}"
+      branch: "autoblock"
```
Comment (DXCanas) [inline .buildkite/pipeline.yml:37]: Will remove this line in subsequent PR once this has been merged

**Comment by DXCanas** [comment]
> is this PR still working in progress? I see that the build got canceled. Thank you!

It had already built and I accidentally  triggered a rebuild 😅 going to let one run anyway

**Review by lyw07 — APPROVED** [review]
this is awesome!

**Pull request merged**